BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/DeleteAccount/DeleteAccountView.swift

```diff
@@ -59,6 +27,85 @@ struct DeleteAccountView: View {
                 store.send(.dismiss)
             }
         }
+        .task {
+            await store.perform(.loadData)
+        }
+    }
+
+    /// This section is shown when the account can be deleted.
+    @ViewBuilder var deleteAccountSection: some View {
+        HStack(spacing: 12) {
+            Image(asset: Asset.Images.warning24)
+                .foregroundColor(Color(asset: Asset.Colors.error))
+
+            VStack(alignment: .leading, spacing: 2) {
+                Text(Localizations.deletingYourAccountIsPermanent)
+                    .foregroundColor(Color(asset: Asset.Colors.error))
+                    .styleGuide(
+                        .headline,
+                        weight: .semibold,
+                        includeLinePadding: false,
+                        includeLineSpacing: false
+                    )
+
+                Text(Localizations.deleteAccountExplanation)
+                    .foregroundColor(Color(asset: Asset.Colors.textSecondary))
+                    .styleGuide(.subheadline)
+            }
+        }
+        .padding(12)
+        .contentBlock()
+
+        VStack(spacing: 12) {
+            AsyncButton(Localizations.deleteAccount) {
+                await store.perform(.deleteAccount)
+            }
+            .buttonStyle(.primary(isDestructive: true))
+            .accessibilityIdentifier("DELETE ACCOUNT")
+
+            Button {
+                store.send(.dismiss)
+            } label: {
+                Text(Localizations.cancel)
+            }
+            .buttonStyle(.secondary(isDestructive: true))
+            .accessibilityIdentifier("CANCEL")
+        }
+    }
+
+    /// This section is shown when the account can't be deleted.
+    @ViewBuilder var preventAccountDeletionSection: some View {
+        HStack(spacing: 12) {
+            Image(asset: Asset.Images.circleX16)
+                .foregroundColor(Color(asset: Asset.Colors.error))
```

Comment:
🤔 I would use `decorative` so SwiftUI ignores the image for accessibility purposes.
```suggestion
            Image(decorative: asset: Asset.Images.circleX16)
                .foregroundColor(Color(asset: Asset.Colors.error))
```